x-pack/plugins/cases/server/client/cases/update.ts

```diff
@@ -337,12 +350,58 @@ async function updateAlerts({
   await casesClientInternal.alerts.updateStatus({ alerts: alertsToUpdate });
 }
 
+function partitionPatchRequest(
```

Comment:
Extracted this logic into a function that creates the different arrays for errored cases and cases that we need to authorize.